app/src/main/res/layout/list_bp_history_item.xml

```diff
@@ -62,6 +64,18 @@
     app:layout_constraintTop_toBottomOf="@id/topGuideline"
     tools:text="120 / 80" />
 
+  <TextView
+    android:id="@+id/bpHigh"
```

Comment:
Can you follow the convention of adding the type to the id as a suffix, please? For ex: `bpHighTextView` or `bpHighLabel`.